php/get_user_watchlist_feed.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
This file should not be here:)

Reply:
actually this file was already there in module.json. So do you want me to remove this file from module.json?

Reply:
Delete this file as well

Reply:
I have deleted this file from my pull request.